### ISSUE TITLE
Repository links in the student view appear to be broken

### ISSUE THREAD

**Issue by 0xf8f8ff** [quickfeed/quickfeed#941]
It looks like the student view repository links to the `Course info`, 'Assignments', and `-labs` repositories aren't working anymore.

**Comment by 0xf8f8ff** [comment]
The reason for this:

Backend makes `Repositries.URLs` (`map[string]string`) with `urls[repoType.String()] = repo.GetHTMLURL()`. 
Frontend receives this in the following format:

```
repos: 

["INFO"]: <URL>,
["ASSIGNMENTS"]: <URL>,
["USER"]: <URL>
```

However, it accesses the map with, for example, `repo[Repository_Type.INFO]}` and so on, which is repo[1] and does not exist in the map.

**Comment by meling** [comment]
Seems like the appropriate fix for this would be something like:
```proto
message Repositories {
    map<Repository.Type, string> URLs = 1;
}
```
I guess this issue could be fixed in #942 as well, since they are related.

**Comment by 0xf8f8ff** [comment]
> ```proto
> map<Repository.Type, string> URLs = 1;
> ```

`Repository.Type` won't compile because it is Enum, but, for example, `uint32` works just fine.
